### ISSUE TITLE
\class after \brief doesn't count as a new line - resulting docs show merged descriptions

### ISSUE THREAD

**Issue by TheMaverickProgrammer** [doxygen/doxygen#6945]
See [this post on SO.](https://stackoverflow.com/questions/55844783/doxygen-detailed-description-bleeds-into-the-brief-description?noredirect=1#comment98352341_55844783)

Consider the following:

```
/*! \brief Spawns two mettaurs on the field
 *  \class TwoMettaurMob
 * 
 * May optionally spawn an empty tile for challenge
 */
```

The output description will be

`Spawns two mettaurs on the fieldMay optionally spawn an empty tile for challenge`

The solution is to add space between the `\brief` and `\class` command

**Comment by albert-github** [comment]
Besides the `\class` command, I think, also the following commands could be set to terminate a brief description:
- `\fn`
- `\var`
- `\typedef`
- `\property`
- `\protocol`
- `\enum`
- `\category`
- `\union`
- `\struct`
- `\namespace`
- `\package`
- `\dir`
- `\def`
- `\defgroup`
- `\idlexcepte`
- `\interface`

and to be consistent with the `\section` command:
- `\page`
- `\mainpage`
- `\file`

List is based on this issue and friend command as well as on the descriptions starting with: "Indicates that a comment block contains documentation..."

@doxygen your opinion ?

**Comment by jwakely** [comment]
The "enhancement" label suggests there's no bug, but I think the missing whitespace between the brief description and the rest of the text is a real bug:

    Spawns two mettaurs on the fieldMay optionally spawn an empty tile for challenge

There should _at least_ be a space between "field" and "May".

**Comment by doxygen** [comment]
> Besides the `\class` command, I think, also the following commands could be set to terminate a brief description:
> 
>     * `\fn`
> 
>     * `\var`
> 
>     * `\typedef`
> 
>     * `\property`
> 
>     * `\protocol`
> 
>     * `\enum`
> 
>     * `\category`
> 
>     * `\union`
> 
>     * `\struct`
> 
>     * `\namespace`
> 
>     * `\package`
> 
>     * `\dir`
> 
>     * `\def`
> 
>     * `\defgroup`
> 
>     * `\ingroup`
> 
>     * `\weakgroup`
> 
>     * `\idlexcepte`
> 
>     * `\interface`
> 
> 
> and to be consistent with the `\section` command:
> 
>     * `\page`
> 
>     * `\mainpage`
> 
>     * `\file`
> 
> 
> List is based on this issue and friend command as well as on the descriptions starting with: "Indicates that a comment block contains documentation..."
> 
> @doxygen your opinion ?

I agree with this.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #6971

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
